<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>coloursConfig</class>
 <widget class="QWidget" name="coloursConfig">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>399</width>
    <height>418</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Colors</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="5">
    <widget class="QLabel" name="label_5">
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Background</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>8</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="2" colspan="3">
    <widget class="QLabel" name="lblSwapUsed_2">
     <property name="text">
      <string>Color:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="buddy">
      <cstring>kcbFreeResourceColour</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="5">
    <widget class="KColorButton" name="kcbFreeResourceColour">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <spacer name="verticalSpacer_3">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>5</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="2" colspan="3">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Translucency:</string>
     </property>
     <property name="buddy">
      <cstring>hsldrFreeResourceTransparency</cstring>
     </property>
    </widget>
   </item>
   <item row="2" column="5" colspan="2">
    <widget class="QSlider" name="hsldrFreeResourceTransparency">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="maximum">
      <number>255</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="QLabel" name="label_6">
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>CPU</string>
     </property>
    </widget>
   </item>
   <item row="4" column="4">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Nice:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="buddy">
      <cstring>kcbCpuNice</cstring>
     </property>
    </widget>
   </item>
   <item row="4" column="5">
    <widget class="KColorButton" name="kcbCpuNice">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="color">
      <color>
       <red>255</red>
       <green>255</green>
       <blue>0</blue>
      </color>
     </property>
     <property name="defaultColor">
      <color>
       <red>255</red>
       <green>255</green>
       <blue>0</blue>
      </color>
     </property>
    </widget>
   </item>
   <item row="5" column="3" colspan="2">
    <widget class="QLabel" name="lblUser">
     <property name="text">
      <string>User:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="buddy">
      <cstring>kcbCpuUser</cstring>
     </property>
    </widget>
   </item>
   <item row="5" column="5" rowspan="2">
    <widget class="KColorButton" name="kcbCpuUser">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="color">
      <color>
       <red>0</red>
       <green>0</green>
       <blue>255</blue>
      </color>
     </property>
    </widget>
   </item>
   <item row="6" column="7" rowspan="2">
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>163</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="7" column="3" colspan="2">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>IOWait:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="buddy">
      <cstring>kcbCpuDisk</cstring>
     </property>
    </widget>
   </item>
   <item row="7" column="5">
    <widget class="KColorButton" name="kcbCpuDisk">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="color">
      <color>
       <red>0</red>
       <green>100</green>
       <blue>0</blue>
      </color>
     </property>
     <property name="defaultColor">
      <color>
       <red>0</red>
       <green>100</green>
       <blue>0</blue>
      </color>
     </property>
    </widget>
   </item>
   <item row="8" column="3" colspan="2">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>System:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="buddy">
      <cstring>kcbCpuSystem</cstring>
     </property>
    </widget>
   </item>
   <item row="8" column="5">
    <widget class="KColorButton" name="kcbCpuSystem">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="color">
      <color>
       <red>255</red>
       <green>0</green>
       <blue>0</blue>
      </color>
     </property>
     <property name="defaultColor">
      <color>
       <red>255</red>
       <green>0</green>
       <blue>0</blue>
      </color>
     </property>
    </widget>
   </item>
   <item row="9" column="1" colspan="2">
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>5</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="10" column="0" colspan="4">
    <widget class="QLabel" name="label_8">
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Memory</string>
     </property>
    </widget>
   </item>
   <item row="11" column="3" colspan="2">
    <widget class="QLabel" name="label_9">
     <property name="text">
      <string>Buffers:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="buddy">
      <cstring>kcbRamBuffers</cstring>
     </property>
    </widget>
   </item>
   <item row="11" column="5">
    <widget class="KColorButton" name="kcbRamBuffers">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="color">
      <color>
       <red>0</red>
       <green>100</green>
       <blue>0</blue>
      </color>
     </property>
     <property name="defaultColor">
      <color>
       <red>0</red>
       <green>100</green>
       <blue>0</blue>
      </color>
     </property>
    </widget>
   </item>
   <item row="12" column="3" colspan="2">
    <widget class="QLabel" name="label_10">
     <property name="text">
      <string>Cached:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="buddy">
      <cstring>kcbRamCached</cstring>
     </property>
    </widget>
   </item>
   <item row="12" column="5">
    <widget class="KColorButton" name="kcbRamCached">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="color">
      <color>
       <red>255</red>
       <green>255</green>
       <blue>0</blue>
      </color>
     </property>
     <property name="defaultColor">
      <color>
       <red>255</red>
       <green>255</green>
       <blue>0</blue>
      </color>
     </property>
    </widget>
   </item>
   <item row="13" column="4">
    <widget class="QLabel" name="label_7">
     <property name="text">
      <string>Used:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="buddy">
      <cstring>kcbRamUser</cstring>
     </property>
    </widget>
   </item>
   <item row="13" column="5">
    <widget class="KColorButton" name="kcbRamUser">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="color">
      <color>
       <red>0</red>
       <green>0</green>
       <blue>255</blue>
      </color>
     </property>
     <property name="defaultColor">
      <color>
       <red>0</red>
       <green>0</green>
       <blue>255</blue>
      </color>
     </property>
    </widget>
   </item>
   <item row="14" column="0" colspan="2">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>5</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="15" column="0" colspan="3">
    <widget class="QLabel" name="label_11">
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Swap</string>
     </property>
    </widget>
   </item>
   <item row="16" column="4">
    <widget class="QLabel" name="lblSwapUsed">
     <property name="text">
      <string>Used:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="buddy">
      <cstring>kcbSwap</cstring>
     </property>
    </widget>
   </item>
   <item row="16" column="5">
    <widget class="KColorButton" name="kcbSwap">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="color">
      <color>
       <red>0</red>
       <green>205</green>
       <blue>205</blue>
      </color>
     </property>
     <property name="defaultColor">
      <color>
       <red>0</red>
       <green>205</green>
       <blue>205</blue>
      </color>
     </property>
    </widget>
   </item>
   <item row="17" column="6">
    <spacer name="verticalSpacer_4">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>16</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KColorButton</class>
   <extends>QPushButton</extends>
   <header>kcolorbutton.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
